tests/workloads/vllm/test_command_gen_strategy_slurm.py

```
@@ -560,34 +365,65 @@ def test_custom_bash_regex_can_target_disaggregated_commands(
+        self, vllm_disagg_tr: TestRun, slurm_system: SlurmSystem
+    ) -> None:
+        tdef = cast(VllmTestDefinition, vllm_disagg_tr.test)
+        assert tdef.cmd_args.prefill is not None
+        tdef.cmd_args.prefill.num_nodes = 2
+        tdef.cmd_args.decode.num_nodes = 2
+        tdef.cmd_args.prefill.ray_head = VllmRayStartArgs.model_validate(
+            {
+                "head": False,
+                "port": 9123,
+                "num_gpus": 4,
+                "dashboard_host": "0.0.0.0",
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)